react-app/nginx.conf

```diff
@@ -1,4 +1,26 @@
 server {
+  # Enable HTTP Strict Transport Security (HSTS) to force clients to always
+  # connect via HTTPS (do not use if only testing)
+  add_header Strict-Transport-Security "max-age=31536000;";
+
+  # Enable cross-site filter (XSS) and tell browser to block detected attacks
+  add_header X-XSS-Protection "1; mode=block";
+
+  # Prevent some browsers from MIME-sniffing a response away from the declared Content-Type
+  add_header X-Content-Type-Options "nosniff";
+
+  # Disallow the site to be rendered within a frame (clickjacking protection)
+  add_header X-Frame-Options "DENY";
+
+  add_header Permissions-Policy "camera=(), microphone=(), geolocation=(), fullscreen=()";
+
+  # Turn off all caddy caching
+  add_header Cache-Control "no-cache,no-store,must-revalidate";
+  add_header Pragma "no-cache";
+
+  # Content Security Policy
+  add_header Content-Security-Policy "default-src 'none'; frame-src 'self' *.gov.bc.ca; script-src 'nonce-windowsp' 'self' *.gov.bc.ca https://maps.googleapis.com https://unpkg.com; style-src 'self' 'unsafe-inline' https://unpkg.com https://cdnjs.cloudflare.com; font-src 'self' *.gov.bc.ca; img-src 'self' *.gov.bc.ca data: tile.openstreetmap.org https://maps.googleapis.com https://unpkg.com https://cdnjs.cloudflare.com https://openmaps.gov.bc.ca; connect-src 'self' *.gov.bc.ca https://openmaps.gov.bc.ca; manifest-src 'self';";
```

Comment:
Where is the maps.googleapis.com used?

Reply:
Good catch, that was probably copied and pasted from the other repo.